### PR TITLE
hoc2015: newer levels.

### PR DESCRIPTION
**Pull request by breville**
After yesterday's classroom visit we have simplified some things (notably removing win/lose game from most levels), and have placed more emphasis on levels with moving characters that are fun to play.  This change updates level naming, ordering, instructions, starter blocks, and toolbox blocks, as well as a few APIs that are used.

### CODE REVIEW

File: apps/src/studio/levels.js [hunk 1]
```diff
@@ -2643,15 +2579,15 @@ levels.hoc2015_blockly_10 = utils.extend(levels.js_hoc2015_event_score_points,
   editCode: false,
```
Comment (cpirich) [inline apps/src/studio/levels.js:2582]: This line needs to change to `js_hoc2015_win_lose` to stay in sync with hoc2015.script

**Comment by cpirich** [comment]
LGTM with one issue that needs be corrected - marked above.
Are we planning to use goUp/goDown in just the maze levels or all levels?
Did you intentionally remove instructions2 from many of the early levels?

I'd prefer you merge this ASAP so we can stay in sync. Integrate frequently and reduce long term pain :smile:

**Comment by breville** [comment]
Thanks Chris, fully agreed on fast merge.  I need to add some rudimentary success criteria before I do that, planned by end of day today, since this build is getting play tested by content partner from time to time.  Is that soon enough?

The idea was to use moveUp in levels 1-6, and then switch to goUp in levels 7-15, to help students differentiate between the grid moving and the event-driven moves.

And yes, the spreadsheet now says no instructions2 in levels 1-6.

**Comment by cpirich** [comment]
It's fine to wait until end of today, but it would be easier for me if you merged now and again at the end of the day. It isn't hurting anyone to have missing success criteria between now and then.

**Pull request merged**